python/tests/test_ops.py

```diff
@@ -1402,6 +1402,27 @@ def test_stack(self):
         np_c = np.stack([np_a, np_b], axis=1)
         self.assertTrue(np.array_equal(c, np_c))
 
+    def test_linspace(self):
+        # Test default num = 50
+        a = mx.linspace(0, 1)
+        np_a = np.linspace(0, 1)
+        self.assertTrue(np.array_equal(a, np_a))
+
+        # Test int32 dtype
+        b = mx.linspace(0, 10, 5, dtype=mx.int32)
+        np_b = np.linspace(0, 10, 5, dtype=int)
+        self.assertTrue(np.array_equal(b, np_b))
```

Comment:
can you utilize, `self.assertEqualArray(b, mx.array(np_b))`
It will check
- shape
- dtype
- correctness of result with options for atol/rtol